src/qibolab/platforms/multiqubit.py

```diff
@@ -41,7 +48,6 @@ def __init__(self, name, runcard):
                             self.qubit_instrument_map[qubit][self.qubit_channel_map[qubit].index(channel)] = name
 
     def reload_settings(self):
```

Comment:
This is still needed for `Multiqubit`, right?

Reply:
My idea is that #463 (which removes `MultiqubitPlatform`) is merged before this so the multiqubit file will not exist anymore here after I fix conflicts.

Reply:
> Is `Multiqubit` actually working in `main`?

Based on today's meeting, I am not sure what is the answer to this question and I cannot check myself as I am not sure which queues have qblox and are operational. Another question is if anyone is using `Multiqubit` from main, and to that I believe the answer is no, because qblox sweepers are only on #463.

> If it isn't, I would drop immediately, and attempt to merge. Then we will merge #463 as soon as it will be ready.

I was avoiding this because doing so will probably create big conflicts with #463 and delay its merge even more, which is bad considering that the end goal is to have **all** instruments operational through the same `Platform` on main.
On the other hand, it is not the first time we are fixing conflicts. So I would say, I am postponing this a bit until I do some other things (hoping #463 progresses) and I will come back to it soon.

Reply:
I am fine waiting for #463, if that is moving towards completion, and the estimate for the time needed is measured in days.

If instead we know for sure that it will require no less than two weeks, maybe it's worth trying to merge this as well, and paying the price of merging once more.

I agree that the goal is to have them *all*, but that one is hard to speed up from my side (same for Stefano, that's why he asked explicitly if we could provide more manpower to help)